tools/build-gdb.sh

```diff
@@ -15,7 +15,7 @@ if [ -z "${N64_INST-}" ]; then
 fi
 
 # Dependency source libs (Versions)
-GDB_V=${GDB_V:-"16.2"}
+GDB_V=16.2
```

Comment:
Do we want to explicitly lock the version here?

Reply:
This is what we do for GCC and friends. I changed to a hard-coded string because of the version number grep in the GitHub Actions workflow:
https://github.com/DragonMinded/libdragon/pull/679/files#diff-be00bc5292287b3090bfd3ce8babaa79307d5539139221a6243bbdf32c4852ccR96

Reply:
yeah I agree we should hardcode it, and then periodically bump it as we do with all other parts of the toolchain

Reply:
That is, interesting :)